libpod/oci.go

```diff
@@ -766,3 +770,15 @@ func (r *OCIRuntime) execStopContainer(ctr *Container, timeout uint) error {
 
 	return nil
 }
+
+// checkpointContainer checkpoints the given container
+func (r *OCIRuntime) checkpointContainer(ctr *Container) error {
+	// imagePath is used by CRIU to store the actual checkpoint files
+	imagePath := filepath.Join(ctr.bundlePath(), "checkpoint")
```

Comment:
If we're going to use this more than once, maybe a helper function for getting it? `ctr.getCheckpointPath()` or similar?

Reply:
will be in next PR update